docs/feature-gates.md

```diff
@@ -330,3 +331,15 @@ disabled. As of now, this feature requires that `AntreaProxy` and `NetworkPolicy
 
 OVS is required to be installed on the virtual machine or the bare-metal server before running Antrea Agent, and the OVS
 version must be >= 2.13.0.
+
+### SupportBundleCollection
+
+`SupportBundleCollection` feature enables a CRD API for Antrea to collect support bundle files on any Node or
+ExternalNode, and upload to a user defined file server.
+
+More documentation will be coming in the future.
+
+#### Requirements for this Feature
+
+User should provide a file server with this feature, and store its authenticator in a Secret. Antrea Controller is
```

Comment:
authenticator -> authentication credential?

Authenticator should mean some component to implement authentication?

Reply:
updated.